tests/test_translations.py

```diff
@@ -20,4 +20,4 @@ def test_translations(client):
     """
     client = Client(schema)
     response = client.execute(query)
-    assert response['data']
+    assert response
```

Comment:
I will change this back in my next PR after correcting the query to look for the new column name